app/state_machines/workflows/predicates.rb

```diff
@@ -139,6 +139,9 @@ def assigned_team_member_and_case_outside_escalation_period?
     responder_is_member_of_assigned_team? && @kase.outside_escalation_deadline?
   end
 
+  def can_start_to_complain?
+    @kase.offender_sar? && (@kase.already_late? || @kase.current_state == 'closed')
+  end
```

Comment:
Are we sure you can't have a complaint about a case before it's "late"?

Reply:
The rules are based on the discussion in that ticket :) as Alex was thinking whether we should only add this button for closed cases, then recalled Users may raise complaint against open cases and the situation we can think of is due to the timeliness (late),  but actually I don't mind to change it if we get the feedback from users, it is small task anyway